src/clims/api/endpoints/work_definition_details.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import absolute_import
+
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+
+from sentry.api.base import Endpoint, DEFAULT_AUTHENTICATION
+from clims.utils import single_or_default
+from clims.api.serializers.models.work_definition import WorkDefinitionSerializer
+from clims.api.serializers.models.workunit import WorkUnitSerializer
+
+
+class WorkDefinitionDetailsEndpoint(Endpoint):
+    authentication_classes = DEFAULT_AUTHENTICATION
+    permission_classes = (IsAuthenticated, )
+
+    name = 'clims-api-0-work-definition-details'
+
+    def get(self, request, work_definition_id):
+        # TODO: Here we split the key outside of the service. Move it into it.
+        process_definition_key, work_definition_key = work_definition_id.split(':')
+        work_definitions = self.app.workflows.get_work_definitions(
+            process_definition_key, work_definition_key)
+        ret = single_or_default(work_definitions)
+        return Response(WorkDefinitionSerializer(ret).data)
```

Comment:
I agree that the naming is not great. I've added an epic for dealing with this at a later point, but we should fix the things that are confusing as soon as we can (although I think we need to merge this one without further change, unless there's something really wrong :)).

I'm also still wondering if we shouldn't call this something else, e.g. Step or UserTask instead. The problem I see with Step, is that we also have non-usertask steps in workflows, so it may be confusing too.

But if this was renamed either, I think we would need:
* [UserTask/Step]DefinitionSerializer (the definition of how the user task should behave, serialized in a list)
* [UserTask/Step]DefinitionDetailsSerializer (the same, serialized in a details view)

Details I think should always be after the Definition word, because it's used to mean that this is a details entry, but not a list entry.

I think we should take this up when we tackle the next task related to this!